### PR TITLE
add master redirects

### PR DESCRIPTION
**Pull request by Akshay1992Yadav**
## [DIR-2406](https://recoverybrands.atlassian.net/browse/DIR-2406)

[DIR-2406]: https://recoverybrands.atlassian.net/browse/DIR-2406?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: nginx/conf.d/redirects/redirects.conf [hunk 1]
```diff
@@ -4667,3 +4667,7 @@ rewrite ^/listings/va-caribbean-healthcare-system-2114025680/?$ https://$host/lo
 rewrite ^/top-rated-treatment-centers/?$ https://$host/top-rated-rehabs/ permanent;
 #REHCORE-16
 rewrite ^/not-ready-to-talk/?$ https://$host/text-support-sign-up/ permanent;
+
+#DIR-2406
+rewrite ^/treatment/heroin/?$ https://$host/heroin/treatment/ permanent;
+rewrite ^/treatment/fentanyl/?$ https://$host/fentanyl/treatment/ permanent;
```
Comment (misunchoi) [inline nginx/conf.d/redirects/redirects.conf]: based on the task looks like /treatment/fentanyl/ is being redirected to /opioids/fentanyl/treatment/ instead

**Review by misunchoi — CHANGES_REQUESTED** [review]

File: nginx/conf.d/redirects/redirects.conf (hunk 1 above)
Comment (Akshay1992Yadav) [reply nginx/conf.d/redirects/redirects.conf]: yeah you are right!, i missed the complete string : P

Comment (Akshay1992Yadav) [reply nginx/conf.d/redirects/redirects.conf]: hey @misunchoi  is there no staging branch here for QA?

Comment (misunchoi) [reply nginx/conf.d/redirects/redirects.conf]: we can QA this on staging once this is merged to the main branch, we'll just need to redeploy the frontend app on staging to see the updates

**Review by misunchoi — APPROVED** [review]

**Pull request merged**